data/sea/52-psv.h

```diff
@@ -336,6 +344,10 @@ static ierror_loc_t psv_read_buffer (psv_state_t *s, const size_t facts_limit)
                                      , entity_cur
                                      , entity_cur_size
                                      , s->fleet );
+                /* we need to flush the output immediately because it might be the case that
+                   the result for the new entity will need to be flushed to drop_fd, while the output
+                   for the old entity here needs to be flushed to output_fd as usual. */
+                msg = psv_flush_to_output (s);
```

Comment:
should this and the call to `psv_write_outputs` above be assigning to `error` rather than `msg`? and should this only occur if msg/error is null?